tools/mac_setup.sh

```diff
@@ -32,6 +32,18 @@ else
     brew up
 fi
 
+# Fix OpenSSL conflicts
+if brew list openssl@1.1 &>/dev/null; then
+    echo "[ ] Unlinking openssl@1.1 to avoid conflicts"
+    brew unlink openssl@1.1
+fi
+
+if brew list openssl@3 &>/dev/null; then
```

Comment:
You'll want to just remove all this stuff I mean. Editing the mac_setup script isn't related to this PR directly, and the fix should be done separately.